### PR TITLE
Updated getting-started.rst

### PR DESCRIPTION
**Pull request by HexedHero**
Added a new section about "Optimizing The Server" since I feel like its needed
Added a better starting up java arg
Removed mentions of old paperclip because of the rebrand
Updated a link to the builds page and the API link for older builds
Some general stuff

### CODE REVIEW

**Comment by KennyTV** [comment]
I wouldn't put Aikar's flags at the bottom, they were at the right place before. I also doubt that a gig is enough for a modern 1.14 server 👀
Also, you're technically not downloading the actual paper server but paperclip, so that shouldn't be changed either

**Comment by HexedHero** [comment]
I moved it to the bottom because I thought it'd fit better being in the configurable part of the doc instead of just getting the server to run. I changed paperclip to paper because when you used to download paper it would be paperclip.jar but now it's not, just confuses people.

For a vanilla server 1G is okay but your right, maybe adding a note saying you may want to increase it but I wanted to make sure almost everyone can actually run it even with a low ram system e.g people just buying a 1g server for their friends.

File: source/server/getting-started.rst [hunk 1]
```diff
@@ -25,31 +25,44 @@ Migrating From CraftBukkit/Spigot
...
 
 Paper provides runnable server jars directly from our continuous integration
-server `here <https://destroystokyo.com/ci/job/Paper>`_.
+server `here <https://papermc.io/downloads>`_.
 
-Download one of the artifacts (either paperclip.jar or paperclip-####.jar, they
-are the same) and save it to a new directory somewhere on your file system.
+Download one of the verions (**Latest recommended**) (paper-###.jar) and save it to a new directory somewhere on your file system or upload it to the root location of your server.
+
+For older versions of Paper:
```
Comment (zachbr) [inline source/server/getting-started.rst]: This is being put directly on the site, this section should be removed.

File: source/server/getting-started.rst [hunk 2]
```diff
@@ -25,31 +25,44 @@ Migrating From CraftBukkit/Spigot
...
 
 To run the server, simply start it up like any other Java application.
 
 Open your terminal, navigate to the saved location, and then run
-``java -jar paperclip.jar``
-
-For more advanced Java tuning, see `this page <https://mcflags.emc.gs>`_
-
-To configure your server, see the :doc:`../server/configuration` page.
+``java -Xms128m -Xmx1g -jar paper.jar``
```
Comment (zachbr) [inline source/server/getting-started.rst]: Recommended configurations of the server have Xms and Xmx match.

File: source/server/getting-started.rst [hunk 3]
```diff
@@ -25,31 +25,44 @@ Migrating From CraftBukkit/Spigot
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Paper is a drop in replacement for both CraftBukkit and Spigot, you don't need
-to make any changes.
+to make any changes apart from the new Paper.yml from us which includes more configurable settings!
```
Comment (zachbr) [inline source/server/getting-started.rst]: You don't _need_ to make any changes, you _can_ but you don't _need_ to.
Drop this change. It is covered below anyway.

**Pull request closed without merging**